BUILDING.md

```diff
@@ -0,0 +1,249 @@
+| audacity_use_lv2                  | STRING | local      | Use lv2 library [system (if available), local, off]             |
+| audacity_use_mad                  | STRING | local      | Use mad library [system (if available), local, off]             |
+| audacity_use_midi                 | STRING | local      | Use midi library [system (if available), local, off]            |
+| audacity_use_nyquist              | STRING | local      | Use nyquist library [local, off]                                |
+| audacity_use_ogg                  | STRING | local      | Use ogg library [system (if available), local, off]             |
+| audacity_use_pa_alsa              | BOOL   | YES        | Use the portaudio ALSA interface if available                   |
+| audacity_use_pa_jack              | STRING | linked     | Use the JACK audio interface if available [loaded, linked, off] |
+| audacity_use_pa_oss               | BOOL   | YES        | Use the OSS audio interface if available                        |
+| audacity_use_pch                  | BOOL   | YES        | Use precompiled headers [yes, no]                               |
+| audacity_use_portaudio            | STRING | local      | Use portaudio library [local]                                   |
+| audacity_use_portmixer            | STRING | local      | Use portmixer library [local, off]                              |
+| audacity_use_portsmf              | STRING | local      | Use portsmf library [system (if available), local, off]         |
+| audacity_use_sbsms                | STRING | local      | Use sbsms library [system (if available), local, off]           |
+| audacity_use_sndfile              | STRING | local      | Use sndfile library [system (if available), local]              |
+| audacity_use_soundtouch           | STRING | local      | Use soundtouch library [system (if available), local, off]      |
+| audacity_use_soxr                 | STRING | local      | Use soxr library [system (if available), local]                 |
+| audacity_use_sqlite               | STRING | local      | Use sqlite library [system (if available), local]               |
+| audacity_use_twolame              | STRING | local      | Use twolame library [system (if available), local, off]         |
+| audacity_use_vamp                 | STRING | local      | Use vamp library [system (if available), local, off]            |
+| audacity_use_vorbis               | STRING | local      | Use vorbis library [system (if available), local, off]          |
+| audacity_use_vst                  | BOOL   | ON         | Use VST2 plug-in support [on, off]                              |
+| audacity_use_wxwidgets            | STRING | local      | Use wxwidgets library [system (if available), local, off]       |
+| audacity_use_zlib                 | STRING | system     | Use zlib library [system (if available), local, off]            |
```

Comment:
I see `buildsystem=1` listed in the examples above but it is not documented here. What does that option do?

Reply:
This activates so-called "Legacy Build System" in XCode. Their new system is very confusing and we currently have odd issues with it.